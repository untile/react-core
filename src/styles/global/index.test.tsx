/**
 * Module dependencies.
 */

import { GlobalStyle } from '.';
import { describe, expect, it } from 'vitest';
import { render } from '@testing-library/react';
import React from 'react';

/**
 * Test `Global` component.
 */

describe(`'Global' component`, () => {
  it('matches snapshot', () => {
    const { asFragment } = render(<GlobalStyle />);

    expect(asFragment()).toMatchSnapshot();
  });
});
