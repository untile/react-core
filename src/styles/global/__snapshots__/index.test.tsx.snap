// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`'Global' component > matches snapshot 1`] = `<DocumentFragment />`;
